stretch_navigation/README.md

```diff
@@ -2,65 +2,61 @@
 
 ## Overview
 
-*stretch_navigation* provides the standard ROS navigation stack as two launch files. This package utilizes gmapping, move_base, and AMCL to drive the stretch RE1 around a mapped space. Running this code will require the robot to be untethered.
+The *stretch_navigation* package provides the standard ROS 2 navigation stack (Nav2) as three launch files. This package utilizes slam_toolbox and Nav2 to drive Stretch around a mapped space. Running this code will require the robot to be untethered.
 
-## Setup
+## Quickstart
 
-These set up instructions are already performed on Stretch RE1 robots. Follow these instructions if *stretch_navigation* is not present in your ROS workspace or you are simulating Stretch on external hardware. Clone stretch_ros to your catkin workspace. Then install dependencies and build the packages, with the following set of commands:
+The first step is to map the space that the robot will navigate in. The `mapping.launch.py` will enable you to do this. First run:
 
 ```bash
-mkdir -p ~/catkin_ws/src
-cd ~/catkin_ws/src
-git clone https://github.com/hello-robot/stretch_ros
-cd ~/catkin_ws
-rosdep install --from-paths src --ignore-src -r -y
-catkin_make
+ros2 launch stretch_navigation mapping.launch.py
 ```
 
-## Quickstart
-
-The first step is to map the space that the robot will navigate in. The `mapping.launch` will enable you to do this. First run:
+Rviz will show the robot and the map that is being constructed. With the terminal open, use the joystick or a keyboard to teleoperate the robot around. Avoid sharp turns and revisit previously visited spots to form loop closures. In Rviz, once you see a map that has reconstructed the space well enough, you can run the following commands to save the map to `stretch_navigation/` and `stretch_user/`.
 
 ```bash
-roslaunch stretch_navigation mapping.launch
+mkdir ~/ament_ws/src/stretch_ros2/stretch_navigation/maps
```

Comment:
Maps should be saved to the stretch_user folder. [See here how maps were saved in Nav1](https://github.com/hello-robot/stretch_ros/tree/dev/noetic/stretch_navigation#quickstart).

Reply:
I think it's more convenient to save a copy of the assets like maps related to navigation in the stretch_navigation package for better accessibility. If they wish to back up maps and use them across distros, they can copy the same in stretch_user folder, as clarified in the tutorials. 

It's a minor improvement, but let me know if it's better to keep this consistent with Nav1 instead.